src/libs/events.js

```diff
@@ -130,10 +131,11 @@ export const extractWorkspaceDetails = (workspaceObject) => {
   // containing the workspace details.
   const workspaceDetails = 'workspace' in workspaceObject ? workspaceObject.workspace : workspaceObject;
   const { name, namespace, cloudPlatform } = workspaceDetails;
+  const hasProtectedData = containsProtectedDataPolicy(workspaceObject.policies ?? workspaceDetails.policies);
```

Comment:
For convenience, this function accepts either a WorkspaceWrapper or the inner WorkspaceInfo object (`workspace.workspace`). However, policies are on the outer WorkspaceWrapper, so this will only work if a WorkspaceWrapper is passed.